_posts/en/newsletters/2022-10-19-newsletter.md

```diff
@@ -0,0 +1,311 @@
+    cleanup] suggested permanently fixing the problem by disallowing any
+    transactions less than 65 bytes in size from being included in new
+    blocks.
+
+    Sanders suggests lowering the transaction relay policy limit from 85
+    bytes to the 65 byte limit suggested in consensus cleanup, which may
+    allow additional experimentation and usage without changing the
+    current risk profile.  Sanders has a [pull request][bitcoin core
+    #26265] open to make this change.  See also [Newsletter #99][news99
+    min] for prior discussion related to this proposed change.
+
+- **BIP324 update:** Dhruv M [posted][dhruv 324] to the Bitcoin-Dev
+  mailing list a summary of several updates to the BIP324 proposal for a
+  [version 2 encrypted P2P transport protocol][topic v2 p2p transport].
+  This includes a rewrite of the [draft BIP][bips #1378] and the
+  publication of a [variety of resources][bip324.com] to help reviewers
+  evaluate the proposal, including an excellent [guide to the proposed
+  code changes][bip324 changes] across multiple repositories.
+
+    As described in the draft BIP's *motivation* section, a native
+    encrypted transport protocol for Bitcoin nodes can improve privacy
+    during transaction announcement, prevent tampering with connections
+    (or at least make it easier to detect tampering), and also make P2P
+    connection censorship and [eclipse attacks][topic eclipse attacks]
+    more difficult.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+<!-- FIXME:harding to update Tuesday -->
+
+- [Bitcoin Core 24.0 RC1][] is the first release candidate for the
+  next version of the network's most widely used full node
+  implementation.  A [guide to testing][bcc testing] is available.
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **btcd v0.23.2 released:**
+  btcd v0.23.2 (and [v0.23.1][btcd 0.23.1]) adds [addr v2][topic addr v2], and additional
+  support for [PSBTs][topic psbt], [taproot][topic taproot], and [MuSig2][topic
+  musig] as well as other enhancements and fixes.
+
+- **ZEBEDEE announces hosted channel libraries:**
+  In a recent [blog post][zbd nbd], ZEBEDEE announced an open source wallet (Open
+  Bitcoin Wallet), Core Lightning plugin (Poncho), Lightning client (Cliché),
+  and Lightning library (Immortan) which focus on support for [hosted channels][].
+
+- **Cashu launches with Lightning support:**
+  E-cash software [Cashu][cashu github] launches as a proof-of-concept wallet with
+  Lightning receive support.
+
+- **Address explorer Spiral launches:**
+  [Spiral][spiral explorer] is a public address [explorer][topic block explorers] that uses
+  cryptography to provide privacy to users querying information about an address.
```

Comment:
I was really skeptical about this, but it looks reasonably legit to me.  (And it's something I've wanted to see for years, I'm excited!)  Thanks @bitschmidty!  I'm going to edit this description to add the phrase "open source", as it took me a bit of effort to find their public code (and without public code, I would not have believed their claim at all).